ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -36,11 +36,10 @@ Item {
 
     property bool isSectionActive: mainModule.activeSection.id === parentModule.getMySectionId()
     property string activeChatId: parentModule && parentModule.activeItem.id
-    property string activeSubItemId: parentModule && parentModule.activeItem.activeSubItem.id
     property int chatsCount: parentModule && parentModule.model ? parentModule.model.count : 0
     property string activeChatType: parentModule && parentModule.activeItem.type
     property bool stickersLoaded: false
-    property var contactDetails: Utils.getContactDetailsAsJson(root.activeChatId, false)
+    property var contactDetails: activeChatType === Constants.chatType.oneToOne && Utils.getContactDetailsAsJson(root.activeChatId, false)
```

Comment:
`activeChatType` above is declared as string, how does this work?

Reply:
Shouldn't it be an `int`?

Reply:
Hmmm, you're right, how did this every work???

Reply:
No idea :shrug:

Reply:
Fixed